developer-docs/how-to-guide/How_to_upload_existing_content_using_sunbird.md

```diff
@@ -10,285 +10,181 @@ allowSearch: true
-    <td>multipart, form-data</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td>X-Authenticated-Userid</td>
-    <td>String</td>
-    <td>Represents the registered User ID authorized to execute the API.</td>
-    <td>userName</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td> Authorization</td>
-    <td>String</td>
-    <td>To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received. </td>
-    <td>api_key</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td>ts</td>
-    <td>String</td>
-    <td>Time Stamp at which upload request was sent.</td>
-    <td>2018-09-05T10:55:22Z</td>
-    <td>Yes</td>
-  </tr>
-</table>
+| Header Parameter       | Parameter Type | Description                                                                                                                                                                       | Sample Value         | Is Mandatory |
+|------------------------|----------------|-------------------------------------------|----------------------|---------------|
+| Content-Type           | String         | The content type entity is the media type of the resource. Possible media types are multipart or form-data                                                                        | multipart, form-data | Yes          |
+| X-Authenticated-Userid | String         | Represents the registered User ID authorized to execute the API.                                                                                                                  | userName             | Yes          |
+| Authorization          | String         | To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received. | api_key              | Yes          |
+| ts                     | String         | Time Stamp at which upload request was sent.                                                                                                                                      | 2018-09-05T10:55:22Z | Yes          |
+
+#### Path Parameters ####
+
+| Path Parameter | Parameter Type | Description                   | Sample Value            | Is Mandatory |
+|----------------|----------------|-------------------------------|-------------------------|--------------|
+| Content ID     | String         | Content ID of the Request URL | do_11258058227713638416 | Yes          |
+
+
+#### Form Data Parameters ####
+
+| Form Data Parameter | Parameter Type | Description                                          | Sample Value | Is Mandatory |
+|---------------------|----------------|------------------------------------------------------|--------------|--------------|
+| file                | String         | The name of the content file that you want to upload | sample.pdf   | Yes          |
```

Comment:
Please add a sample request which is motivated by the scenario. Please explain how the contentId and ContentType parameters are set in the request

Reply:
@kochhar Done as per discussion with Mahesh.